### PR TITLE
Incorporating feedback on documentation and order of videos

### PR DESCRIPTION
**Pull request by kul3r4**

### CODE REVIEW

**Comment by kul3r4** [comment]
I signed it!

2016-03-14 17:55 GMT+00:00 googlebot <notifications@github.com>:

> Thanks for your pull request. It looks like this may be your first
> contribution to a Google open source project. Before we can look at your
> pull request, you'll need to sign a Contributor License Agreement (CLA).
>
> [image: :memo:] *Please visit https://cla.developers.google.com/
> <https://cla.developers.google.com/> to sign.*
>
> Once you've signed, please reply here (e.g. I signed it!) and we'll
> verify. Thanks.
> ------------------------------
>
>    - If you've already signed a CLA, it's possible we don't have your
>    GitHub username or you're using a different email address. Check your
>    existing CLA data <https://cla.developers.google.com/clas> and verify
>    that your email is set on your git commits
>    <https://help.github.com/articles/setting-your-email-in-git/>.
>    - If you signed the CLA as a corporation, please let us know the
>    company's name.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ampproject/amp-by-example/pull/84#issuecomment-196439918>
> .
>



-- 
Chiara Chiappini

File: src/20_Components/amp-video.html [hunk 1]
```diff
@@ -19,7 +19,10 @@
 
   <!-- #### Basic Usage -->
   <!--
-    A simple video with controller. Boolean attribute (`controls` in this case) must be without values.
+    A simple video with controller. 
+    `amp-video` supports the following formats: mp4, webm, ogg together with all the formats
+    supported by the HTML5 video tag. HLS is also supported.
```
Comment (sebastianbenz) [inline src/20_Components/amp-video.html]: s/. HLS is also support/including HLS/

**Comment by sebastianbenz** [comment]
* Can you make the commit message more specific.
* If we change the order we have to make it explicit. Can we change the sections to something like:

```
#### Embed your own videos with amp-video
#### Supported Video Platforms
##### amp-vimeo
##### amp-youtube
...
#### What if your video provider is not supported?  
Use an amp-iframe ...
```

**Comment by kul3r4** [comment]
PTAL

File: src/30_Advanced/Integrating_Videos_in_AMP%3A_an_Overview.html [hunk 2]
```diff
@@ -132,6 +122,17 @@
                            layout=responsive
                            data-videoid="lBTCB7yLs8Y" >
   </amp-youtube>
+  <!-- #### What if your video provider is not supported?-->
+  <!-- 
+    Use `amp-iframe` for native videos not supported in AMP.
```
Comment (sebastianbenz) [inline src/30_Advanced/Integrating_Videos_in_AMP%3A_an_Overview.html]: remove native

File: src/30_Advanced/Integrating_Videos_in_AMP%3A_an_Overview.html [hunk 3]
```diff
@@ -27,7 +27,23 @@
 </head>
 <body>
 
-  <!-- #### amp-brightcove -->
+  <!-- #### Embed your own videos with amp-video -->
```
Comment (sebastianbenz) [inline src/30_Advanced/Integrating_Videos_in_AMP%3A_an_Overview.html]: make Video uppercase (title case)

File: src/30_Advanced/Integrating_Videos_in_AMP%3A_an_Overview.html [hunk 4]
```diff
@@ -132,6 +122,17 @@
                            layout=responsive
                            data-videoid="lBTCB7yLs8Y" >
   </amp-youtube>
+  <!-- #### What if your video provider is not supported?-->
```
Comment (sebastianbenz) [inline src/30_Advanced/Integrating_Videos_in_AMP%3A_an_Overview.html]: use also title case "What if you Video Provider is not Supported"

**Comment by kul3r4** [comment]
PTAL

**Comment by sebastianbenz** [comment]
LGTM

**Pull request merged**